TESTING.md

```diff
@@ -119,13 +119,23 @@ Prerequisite: You have to be **signed in**. This is important because we expect
 
 ## Commands
 
-### General commands availability in Cody toolbar
+### General commands availability in Cody tool window
 
-1. Navigate to `Cody` toolbar and open `Commands`.
+1. Navigate to `Cody` tool window and open `Commands`.
 
 #### Expected behaviour
 
-* List of commands is immediately available after the toolbar is displayed. **No refresh is required.**
+* List of commands is immediately available after the tool window is displayed. **No refresh is required.**
+
+### General commands availability from keyboard shortcuts
+
+1. Explain Code: <kbd>control</kbd> + <kbd>Shift</kbd> + <kbd>E</kbd>
+2. Smell Code: <kbd>control</kbd> + <kbd>Shift</kbd> + <kbd>S</kbd>
+3. Generate Test: <kbd>control</kbd> + <kbd>Shift</kbd> + <kbd>T</kbd>
```

Comment:
`control` != `Ctrl`?

Reply:
control works on macOs

I am trying to find a corresponding (or alternative shortcut on windows). Please let me know if you find something on Linux.

Reply:
Thanks! But what's the difference between `control` and `meta`? 

(My confusion arises from the fact that in the Settings, it says 'meta', while in the testing docs, it's 'control' - sorry if I'm asking stupid questions.)

Reply:
`meta` removed, closing thread